smart_meter_texas/__init__.py

```diff
@@ -85,6 +87,55 @@ async def read_meter(self, client: Client):
                 else:
                     _LOGGER.error("Unknown meter reading status: %s", status)
                     raise SmartMeterTexasAPIError(f"Unknown meter status: {status}")
+    async def get_15min(self, client: Client):
+        """Get the interval data to parse out Surplus Generation"""
+        _LOGGER.debug("Getting Interval data")
+        surplus = []
+        yesterday = (datetime.date.today() - datetime.timedelta(days=1)).strftime("%m/%d/%Y")
+
+        json_response = await client.request(
+                INTERVAL_SYNCH,
+                json={"startDate": yesterday,
+                    "endDate": yesterday,
+                    "reportFormat": "JSON",
+                    "ESIID": [self.esiid],
+                    "versionDate": None,
+                    "readDate": None,
+                    "versionNum": None,
+                    "dataType": None
+                },
+            )
+        try:
+            data = json_response["data"]
+            energy = data["energyData"]
+        except KeyError:
+            _LOGGER.error("Error reading data: ", json_response)
+            raise SmartMeterTexasAPIError(
+                    f"Error parsing response: {json_response}"
+                )
+        else:
+            h=-1
+            minute=0
+            for E in energy:
```

Comment:
Please use variables names that meet the [PEP8 style](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).

Reply:
I believe i have resolved this, please let me know if i've missed something.